library/std/src/keyword_docs.rs

```diff
@@ -3,8 +3,8 @@
 /// Cast between types, or rename an import.
 ///
 /// `as` is most commonly used to turn primitive types into other primitive types, but it has other
-/// uses that include turning pointers into addresses, addresses into pointers, and pointers into
-/// other pointers.
+/// uses that include turning pointers into addresses, addresses into pointers, pointers into
+/// other pointers, and qualifying paths for associated items.
```

Comment:
Please revert this since this paragraph is about casting only (with my suggestion above, we would now mention it in the introductory paragraph which seems more suited).